Smash_Combos.Persistence/Migrations/20200816160856_UpdatedForModeration.cs

```diff
@@ -0,0 +1,112 @@
+                    DateInfracted = table.Column<DateTime>(nullable: false)
+                },
+                constraints: table =>
+                {
+                    table.PrimaryKey("PK_Infractions", x => x.Id);
+                    table.ForeignKey(
+                        name: "FK_Infractions_Users_UserId",
+                        column: x => x.UserId,
+                        principalTable: "Users",
+                        principalColumn: "Id",
+                        onDelete: ReferentialAction.Cascade);
+                });
+
+            migrationBuilder.CreateTable(
+                name: "Reports",
+                columns: table => new
+                {
+                    Id = table.Column<int>(nullable: false)
+                        .Annotation("Npgsql:ValueGenerationStrategy", NpgsqlValueGenerationStrategy.IdentityByDefaultColumn),
+                    UserId = table.Column<int>(nullable: false),
+                    ReporterId = table.Column<int>(nullable: false),
+                    Body = table.Column<string>(nullable: false),
+                    DateReported = table.Column<DateTime>(nullable: false),
+                    Dismiss = table.Column<bool>(nullable: false),
+                    ComboId = table.Column<int>(nullable: true),
+                    CommentId = table.Column<int>(nullable: true)
+                },
+                constraints: table =>
+                {
+                    table.PrimaryKey("PK_Reports", x => x.Id);
+                    table.ForeignKey(
+                        name: "FK_Reports_Combos_ComboId",
+                        column: x => x.ComboId,
+                        principalTable: "Combos",
+                        principalColumn: "Id",
+                        onDelete: ReferentialAction.Restrict);
+                    table.ForeignKey(
+                        name: "FK_Reports_Comments_CommentId",
+                        column: x => x.CommentId,
+                        principalTable: "Comments",
+                        principalColumn: "Id",
+                        onDelete: ReferentialAction.Restrict);
+                });
+
+            migrationBuilder.CreateIndex(
+                name: "IX_Infractions_UserId",
+                table: "Infractions",
+                column: "UserId");
+
+            migrationBuilder.CreateIndex(
+                name: "IX_Reports_ComboId",
+                table: "Reports",
+                column: "ComboId");
+
+            migrationBuilder.CreateIndex(
+                name: "IX_Reports_CommentId",
+                table: "Reports",
+                column: "CommentId");
+        }
+
+        protected override void Down(MigrationBuilder migrationBuilder)
+        {
+            migrationBuilder.DropTable(
+                name: "Infractions");
+
+            migrationBuilder.DropTable(
+                name: "Reports");
+
+            migrationBuilder.DropColumn(
```

Comment:
Similar thing to the changes required to migrate up, you should create the column first and migrate admins back